src/main/resources/assets/multiconnect/lang/id_id.json

```diff
@@ -0,0 +1,9 @@
+{
+  "multiconnect.betaWarning.line1": "%s is currently in beta support by multiconnect. %s saat ini saat dalam dukungan beta oleh multiconnect.",
```

Comment:
There's some English left in here

Reply:
Oh, I forgot to delete it. I usually do that to compare it